assembl/static/js/app/views/infobar.js

```diff
@@ -41,6 +41,26 @@ var PiwikIframeModal = Backbone.Modal.extend({
   constructor: function PiwikIframeModal(){
     Backbone.Modal.apply(this, arguments);
   },
+
+  getStatsUrl: function(){
+    // var url = "//piwik.coeus.ca/index.php?module=CoreAdminHome&action=optOut&language=fr";
+    var url = analyticsUrl;
+    if (! (url[url.length-1] === '/')) {
+      url = url + "/";
+    }
+    url = url + "index.php?module=CoreAdminHome&action=optOut&language=";
+    var locale = Ctx.getLocale() || "fr";  // Either get the locale, or the default is French
```

Comment:
Hmmm... with all the cascades in the backend, which end in server settings, I really cannot see a scenario where Ctx.getLocale() would be null. I'd be more comfortable failing all of assembl if it were (I'm sure that would happen anyway.)
Hardcoding a default in the templates is a no-no obviously.